data/data/bootstrap/files/etc/NetworkManager/dispatcher.d/pre-up.d/dns-vip-prepender

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
These files will be created for all platforms... that is probably not correct.

Also detailed description of each file (why, what) in the commit would be highly appreciated.

Reply:
Some files are necessary - during the bootstrap stage I need to start few static containers to make internal DNS and internal API IP visible to all components. 

Where is the right place to add a README for ovirt ?

Reply:
> Where is the right place to add a README for ovirt ?

`docs/dev/ovirt` or `docs/user/ovirt`, depending on intended audience.

Reply:
README is added and more info goes into the linked enhancement